backend/routes/nft.go

```diff
@@ -53,8 +54,18 @@ func getNFT(w http.ResponseWriter, r *http.Request) {
 
 func getMyNFTs(w http.ResponseWriter, r *http.Request) {
 	address := r.URL.Query().Get("address")
+	//Pagination
+
+	pageStr := r.URL.Query().Get("page")
+	limit := 10 // Default limit
```

Comment:
Can you make this a query param with a default limit of 10 and a max value of 25.